docs/technical-design.md

```diff
@@ -0,0 +1,335 @@
+# Technical Design: Testing Drivers Against Atlas Planned Maintenance
+
+This document is **not** updated regularly. It's only purpose is to
+provide the reader with information about design choices and decisions
+made during the conception and initial implementation of the `astrolabe`
+framework. For up-to-date information on how to use `astrolabe`, please
+see the `integration-guide`.
+
+Title  
+Testing Drivers against Atlas Planned Maintenance
+
+Author  
+Prashant Mital
+
+Lead  
+Matt Broadstone
+
+Advisors  
+Andrew Davidson, Sheeri Cabral, Will Shulman, Jeremy Mikola, Ian Whalen,
+Rachelle Palmer
+
+Status  
+Approved
+
+Type  
+Process
+
+Minimum Server Version  
+N/A
+
+Last Modified  
+January 28, 2020
+
+## Abstract
+
+Testing drivers against Atlas clusters undergoing maintenance is
+required to increase the likelihood that deviations of driver behavior
+from the high-availability and/or graceful-failure characteristics
+promised by [driver
+specifications](https://github.com/mongodb/specifications) are detected
+and rectified during development. This specification outlines a testing
+framework that streamlines the process of testing drivers against Atlas
+clusters undergoing maintenance thereby ensuring expeditious
+incorporation of this additional testing into all driver projects. The
+improved test coverage will result in a superior user experience as
+workload disruption during planned maintenance due to undiscovered bugs
+in the drivers, server or Atlas will become less likely.
+
+## Definitions
+
+### META
+
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”,
+“SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this
+document are to be interpreted as described in [RFC
+2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+### Terms
```

Comment:
Looks like this section previously had [indentations](https://github.com/mongodb-labs/drivers-atlas-testing/blob/master/docs/source/technical-design.rst#terms) for the terms?

I think it's fine if we just bolden the word and leave the other part as is.

Reply:
Yeah bold sounds good